app/containers/Bookings/Results/SearchForm.js

```diff
@@ -12,30 +11,62 @@ import './SearchForm.scss'
 
 class SearchAgainForm extends Component {
   constructor(props) {
-    const asArray = data => (data && Array.isArray(data) ? data : [data])
-
+    const asArray = data => {
```

Comment:
You may be able to use Array.from() instead.

Reply:
Don't think so.  Array.from('abc') will return ['a', 'b', 'c'], not ['abc'].
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from